share/multiline.py

```diff
@@ -21,17 +24,17 @@ class CommonMultiline(metaclass=ABCMeta):
     Common class for Multiline components
     """
 
-    _feed: Iterator[tuple[bytes, bytes, int]]
```

Comment:
i think defining a type alias for this iterator would make this more readable (and isolate any changes like this one)